api/v1alpha1/project_types.go

```diff
@@ -87,6 +87,10 @@ type ProjectSpec struct {
 	// The ID of the project's category
 	// +optional
 	CategoryId int `json:"categoryId,omitempty"`
+
+	// The Custermer Access Status of a project
+	// +optional, if not provided default behaviour is False
+	CustomerAccessStatus bool `json:"customerAccessStatus,omitempty"`
```

Comment:
This is not self explanatory, please use a better variable name since this might be confusing for end users